src/mod/server.mod/server.c

```diff
@@ -1422,6 +1423,7 @@ static tcl_ints my_tcl_ints[] = {
   {"stack-limit",       &stack_limit,               0},
   {"exclusive-binds",   &exclusive_binds,           0},
   {"msg-rate",          &msgrate,                   0},
+  {"oident",            &oident,                    0},
```

Comment:
Minor nitpick — it's *oidentd*, even when not referring to the daemon itself.

Reply:
Thank you, done.